abel/tests/test_onion_bordas.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import absolute_import
```

Comment:
Are all of the tests of the transform methods really similar? Perhaps we just need one test file that loops over all of the methods?